### PR TITLE
Setting up release

### PR DESCRIPTION
**Pull request by bhavyalatha26**
### Testing done
As part of the initial plugin release, this PR contains changes recommended as in the [Releasing your plugin](https://jenkins.io/doc/developer/publishing/releasing/) section.
Changes include:
- Incrementals Enablement 
- Configure CD workflow
- Configure Release Drafter
- Configure Dependabot
- Update Maven POM and Config

### Submitter checklist
- [ x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ x] Ensure that the pull request title represents the desired changelog entry
- [ x] Please describe what you did
- [ x] Link to relevant issues in GitHub or Jira
- [ x] Link to relevant pull requests, esp. upstream and downstream changes
- [ x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by bhavyalatha26** [comment]
Thanks for approving the changes @timja. Can we merge the PR ?

**Comment by timja** [comment]
You can see although I would suggest you fix your build, (spotless is failing)

**Comment by bhavyalatha26** [comment]
> You can see although I would suggest you fix your build, (spotless is failing)

Hi @timja fixed the issues, but not sure about the fix for above **stapler warning alert.** Can you please help us with code recommendation for this.

Regards,
Bhavya

File: src/main/java/io/jenkins/plugins/cloudaeye/CloudAEyeGlobalKeyConfiguration.java [hunk 1]
```diff
@@ -86,26 +86,29 @@
      * @throws ServletException
      */
     @POST
-    public FormValidation doTestConnection(@QueryParameter("tenantKey") final Secret tenantKey,
-                                           @QueryParameter("token") final Secret token) throws IOException, ServletException {
+    public FormValidation doTestConnection(
```
[the remark this answers is not in the document]
Comment (bhavyalatha26) [reply src/main/java/io/jenkins/plugins/cloudaeye/CloudAEyeGlobalKeyConfiguration.java:93]: Fixed as per the suggestion

**Comment by bhavyalatha26** [comment]
> You can see although I would suggest you fix your build, (spotless is failing)

Hi @timja thanks for the suggestion, fixed all the related issues. Please let us if the PR can be merged now (please also verify the labels and let us know any new labels needs to be added or the existing one is good enough)

**Comment by timja** [comment]
> > You can see although I would suggest you fix your build, (spotless is failing)
> 
> Hi @timja thanks for the suggestion, fixed all the related issues. Please let us if the PR can be merged now (please also verify the labels and let us know any new labels needs to be added or the existing one is good enough)

Hi no the labels aren't right for CD, please take a look at https://www.jenkins.io/doc/developer/publishing/releasing-cd/#no-release-has-been-triggered-after-merging-a-pull-request

**Comment by bhavyalatha26** [comment]
> > > You can see although I would suggest you fix your build, (spotless is failing)
> > 
> > 
> > Hi @timja thanks for the suggestion, fixed all the related issues. Please let us if the PR can be merged now (please also verify the labels and let us know any new labels needs to be added or the existing one is good enough)
> 
> Hi no the labels aren't right for CD, please take a look at https://www.jenkins.io/doc/developer/publishing/releasing-cd/#no-release-has-been-triggered-after-merging-a-pull-request

Thanks for the recommended links @timja , we applied `major-rfe` label as that was the relevant one in the available list. Please confirm if we can merge the PR.

**Comment by timja** [comment]
its your plugin you can merge when you like.

**Comment by bhavyalatha26** [comment]
> its your plugin you can merge when you like.

Ok thank you !

**Pull request merged**